js/map.mjs

```diff
@@ -147,6 +147,22 @@ async function initMap(){
             mapContainer.style = "top:0;padding:2px;"
         }
     }
+    //Setting parameters
+    let settingsArray = ["Location", "Nirnroot", "Wayshrine", "NearbyGates"];
+    for(let para of settingsArray){
+        if(windowParams.get(para.toLowerCase()) == "true"){
+            document.getElementById("button_" + para).checked = true;
+        }
+        if(windowParams.get(para.toLowerCase()) == "false"){//can't do else for these since it would override user settings.
+            document.getElementById("button_" + para).checked = false;
+        }
+    }
+    //Prediscovered appears to be tied to settings, so I can't figure out how to make that one changable. Most people won't turn it off anyway.
+    let tspSetting = windowParams.get("tsp")
+    if(tspSetting == "none" || tspSetting == "location" || tspSetting == "nirnroot"){
+        document.getElementById("button_tsp" + tspSetting.charAt(0).toUpperCase() + tspSetting.slice(1)).checked = true;
+    }
+
```

Comment:
wondering if we want to shorten the url params (param names or values) in any way.
as specifying them all creates a pretty long url.
IDK.

Reply:
the names are long but idk if the link length really impacts the end user much since they are just pulling these up out of guide instructions and probably not looking at the link itself. I feel like if we add more parameters down the road it's better to have each of them be more descriptive now